pl-l2d43c1/pl-l2d43c1.xml

```diff
@@ -276,47 +276,86 @@
         , de impoenitentia, quae est blasphemia in Spiritum
           sanctum, sic ait: 
         <cit>
+          <quote>
           lsta impoenitentia vel cor impoenitens, quamdiu quisque in hac
           carne vivit, non potest iudicari. De nullo enim desperandum est, quamdiu patientia Dei ad
-          poenitentiam adducit. Paganus est hodie, 
+          poenitentiam adducit. Paganus est hodie,
+          <quote>
```

Comment:
closing slash in close `</quote>` is missing. (You're editor should be giving you red errors when you do this; we can check on Thursday to make sure that is working)